### ISSUE TITLE
v1.3.7 Breaking Windows Store Version

### ISSUE THREAD

**Issue by TarkinMX** [robot9706/FSSE#73]
Using FSSE Win v1.3.7 and it breaks my gamesave on the Windows Store version of the game. Currently have windows v1.9 installed. The editor almost doubles the size of the gamesave and when trying to start the save in the game it causes the game to crash. A current workaround I found is using this editor to make my changes and using another gamesave editor I have called FalloutShelter save editor to re-open and re-save the gamesave which causes the save to go back down near its original size. I'd rather use your editor as seeing it has many more options to choose from.

**Comment by robot9706** [comment]
The file size changes because I serialize the JSON differently to make it easier to debug vaults. But that does not change the JSON data the content is the same it's just in a more human readable format. This shouldn't make the game crash (and it never crashed for me because of vault file size).

I can compress the file like the game does to see if that makes the tool more useful.

**Comment by TarkinMX** [comment]
I was wrong, I appologize. I thought the porblem had something to do with the file size but it was any time I tried deleting multiple Mr. Handys that was causing the problem.

**Comment by robot9706** [comment]
No need to be sorry, bugs are essential part of FSSE. :)
Btw I added the compression which the game does now a save is smaller than the original O.o It seems to be working but it's strange. I'll release 1.3.8 today (hopefully).

But I think you found a different issue with MrHandies. Could you share some details about that?

**Comment by TarkinMX** [comment]
I had maybe 5 Mr. Handies waiting outside my vault door and I wanted to get rid of them but every time I removed them using your save editor and then trying to start the game it would make the game crash when loading the save. The workaround I found was instead of deleting them, I deleted the ones already assigned inside the vault and then loaded the gamesave and moved the ones waiting outside into the vault.
